### PR TITLE
Passthrough following JLab3 update

### PR DESCRIPTION
**Pull request by fcollonval**
- [x] basics/hello-world
- [x] basics/datagrid
- [x] basics/signals
- [x] command-palette
- [x] commands
- [x] context-menu
- [x] launcher
- [x] log-console
- [x] main-menu
- [x] react
- [x] settings
- [x] state
- [x] widget-tracker
- [x] advanced/server-extension
- [x] advanced/kernel-messaging
- [x] advanced/kernel-output

+ Add links checker

### CODE REVIEW

**Comment by fcollonval** [comment]
This is ready for review - the failing check is the infamous timeout on macos

File: advanced/server-extension/README.md [hunk 1]
```diff
@@ -28,38 +28,48 @@ like this (be careful to set _has\_server\_extension_ to _y_):
...
+> The python name should not contain `-`. It is nice for user to test your extension online, so the `has_binder` was set to _yes_.
+
 The cookiecutter creates the directory `jlab_ext_example` [or your extension name]
 that looks like this:
 
 ```bash
 jlab_ext_example/
 │  # Generic Files
 │   .gitignore
+│   install.json                # Information retrieved by JupyterLab to help users
```
Comment (jtpio) [inline advanced/server-extension/README.md:48]:
```suggestion
│   install.json                # Information retrieved by JupyterLab to help users know how to manage the extension
```

File: basics/hello-world/README.md [hunk 2]
```diff
@@ -62,11 +81,17 @@ Those files can be separated in 3 groups:
...
+  - `style/` folder contains style elements that you can use
 - Validation:
   - `.prettierrc` and `.prettierignore` specify the code formatter [`prettier`](https://prettier.io) configuration
   - `.eslintrc.js` and `.eslintignore` specify the code linter [`eslint`](https://eslint.org) configuration
   - `.github/workflows/build.yml` sets the continuous integration tests of the code using [GitHub Actions](https://help.github.com/en/actions)
+- Packaging as a Python package:
+  - `setup.py` contains information about the Python package such as what to package
+  - `pyproject.toml` contains the dependencies to create the Python package
+  - `MANIFEST.in` contains list of non-Python files to include in the Python package
+  - `install.json` contains information retrieved by JupyterLab to help user install the package
```
Comment (jtpio) [inline basics/hello-world/README.md:93]:
```suggestion
  - `install.json` contains information retrieved by JupyterLab to help user know how to manager the package
```

File: basics/hello-world/README.md [hunk 3]
```diff
@@ -103,22 +128,29 @@ of the `JupyterFrontEndPlugin` class:
...
-
-console.log('the JupyterLab main application:', app);
+    console.log('JupyterLab extension hello-world is activated!');
 ```
 
 ```ts
-// src/index.ts#L17-L17
+// src/index.ts#L14-L17
+
+  }
```
Comment (jtpio) [inline basics/hello-world/README.md:152]: Looks like this might require a `embedme` update?

**Comment by jtpio** [comment]
Thanks, looks good!

File: basics/hello-world/README.md (hunk 3 above)
Comment (fcollonval) [reply basics/hello-world/README.md:146]: Thanks for the review.

I corrected the snippet before this one. From the text, the goal is to present the code state generated from the template. So this is split in 3 snippets as the code changes latter in the example.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**